utils/setter.go

```diff
@@ -0,0 +1,43 @@
+package utils
+
+import (
+	"fmt"
+	"reflect"
+	"strconv"
+	"strings"
+)
+
+// SetValue assign value to the field
+// name of the struct dest.
+// Returns an error if name is not found or dest cannot
+// be addressed.
+func SetValue(name string, dest interface{}, value string) error {
+	tmp := reflect.Indirect(reflect.ValueOf(dest)).FieldByNameFunc(func(s string) bool {
+		return strings.ToUpper(s) == strings.ToUpper(name)
+	})
+	if !tmp.IsValid() {
+		return fmt.Errorf("%s was not found in %T", name, dest)
+	}
+	if !tmp.CanAddr() {
+		return fmt.Errorf("%s from %T cannot be addressed", name, dest)
+	}
+	var v interface{}
+	var err error
+	switch k := reflect.TypeOf(tmp.Interface()).Kind(); k {
```

Comment:
please stop doing `reflect.TypeOf(v.Interface())`; just do `v.Type()`

Reply:
sorry, I didn't know about that before the last PR and I missed this one...

Reply:
no worries :)